src/current/v24.1/cockroachdb-feature-availability.md

```diff
@@ -31,54 +31,60 @@ Any feature made available in a phase prior to GA is provided without any warran
 **The following features are in limited access** and are subject to change. To begin validating a limited access feature and share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/hc).
 {{site.data.alerts.end}}
 
-### CockroachDB Dedicated on Azure
+### Export logs to Azure Monitor
```

Comment:
general comment this diff is really hard to review with the new additions plus shuffling sections around the page 

in future it would be easier to review a smaller diff without also moving page elements

moving page elements in a separate PR would be preferred from my POV as a reviewer